# custom messages for publishing and subscribing to visualization data
# of Reachability analysis using Flowstar
#
Header header

int64 output_id		#sequence number of the output
string rectangle_x	# rect x min
string rectangle_x2	# rect x max
string rectangle_y	# rect y min
string rectangle_y2	# rect y max
